soc/arm/silabs_exx32/efr32mg12p/dts_fixup.h

```diff
@@ -9,6 +9,9 @@
 /* SoC level DTS fixup file */
 
 #define DT_NUM_IRQ_PRIO_BITS	DT_ARM_V7M_NVIC_E000E100_ARM_NUM_IRQ_PRIORITY_BITS
+
+#define DT_NUM_MPU_REGIONS      DT_ARM_ARMV7M_MPU_E000ED90_ARM_NUM_MPU_REGIONS
```

Comment:
This must be removed, cause the DT_ define is not generated

Reply:
done and retested, the sanitycheck tests still pass.